From 023b27fd05dbda86679b5e6237939520aca75f53 Mon Sep 17 00:00:00 2001
From: Robert Senderek <robert.senderek@10g.pl>
Date: Tue, 11 Feb 2025 17:11:21 +0100
Subject: [PATCH] tplink-safeloader: Add more special_id's for MR70X

non-EU regions and newer firmware for MR70Xv1 have different special_id
MR1800X is exacly same hw and shares OEM fw. Intened to be use as alt name

scanned files:

MR70Xv1_eu_jp_us_br_eg-up-ver1-1-0-P1[20240311-rel52779]_2024-03-11_14.43.29.bin
MR1800Xv1_eu-up-ver1-1-0-P1[20240311-rel52779]_2024-03-11_14.43.29.bin

Signed-off-by: Robert Senderek <robert.senderek@10g.pl>
---
 src/tplink-safeloader.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/tplink-safeloader.c b/src/tplink-safeloader.c
index 9b2f7d0..aca5346 100644
--- a/src/tplink-safeloader.c
+++ b/src/tplink-safeloader.c
@@ -3383,9 +3383,13 @@ static struct device_info boards[] = {
 		.vendor = "",
 		.support_list =
 			"SupportList:\n"
+			"{product_name:MR1800X,product_ver:1.0.0,special_id:45550000}\n"
 			"{product_name:MR70X,product_ver:1.0.0,special_id:45550000}\n"
 			"{product_name:MR70X,product_ver:1.0.0,special_id:4A500000}\n"
-			"{product_name:MR70X,product_ver:1.0.0,special_id:55530000}\n",
+			"{product_name:MR70X,product_ver:1.0.0,special_id:55530000}\n"
+			"{product_name:MR70X,product_ver:1.0.0,special_id:53470000}\n"
+			"{product_name:MR70X,product_ver:1.0.0,special_id:42520000}\n"
+			"{product_name:MR70X,product_ver:1.0.0,special_id:45470000}\n",
 		.part_trail = 0x00,
 		.soft_ver = SOFT_VER_DEFAULT,
 
